projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD013 -->
+# Transparency of algorithmic decision making
+
+This document contains a checklist with requirements for tools we could use to help with the transparency of algorithmic decision making.
+
+The requirements are based on:
+
+- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a framework for evaluating software quality.
+- **Industry best practices:** This includes a broad range of recommendations and guidelines for IT tool development and implementation.
+- **Common IT tool requirements:** This information was gathered by analyzing various sources, such as documentation from popular IT tools, user reviews, and articles from reputable tech publications that discuss essential features and functionalities expected from different types of IT tools.
+- **Internal discussion and common sense**: While above sources are already exhaustive, we also used team discussions and our own knowledge.
+
+The requirements have been given a priority based on the [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method) scale to allow for tool comparison.
+
+| ID   | Group                  | Requirement                                                  | Priority |
+| ---- | ---------------------- | ------------------------------------------------------------ | -------- |
+| 1... | Functionality          | The tool allows users to perform technical tests on an algorithm or model. | M        |
+|      | Functionality          | The tool allows users to choose which tests to perform.      | M        |
+|      | Functionality          | The tool supports plugin functionality so additional tests can be added easily. | S        |
+|      | Functionality          | The tool can generate a human readable report.               | M        |
+|      | Functionality          | The tool can generate a machine readable report.             | S        |
+|      | Functionality          | The tool allows various formats for reports, like PDF, JSON, MD. | C        |
+|      | Functionality          | The tool allows to create custom reports based on components. | S        |
+|      | Functionality          | It is possible to add custom components for reports.         | S        |
+|      | Functionality          | The tool provides logging.                                   | S        |
+|      | Functionality          | The tool can be used on an isolated system without an internet connection. | S        |
+|      | Functionality          | The tool does not share any data or logging to anyone or anywhere. | C        |
+|      | Functionality          | The tool can be integrated in a CI/CD flow.                  | C        |
+|      | Functionality          | The tool can be offered as a (cloud) service where no local installation is required. | C        |
+|      | Reliability            | The tool can operate consistently and reliably.              | M        |
```

Comment:
this one is a bit vague. Could you add what consistent and reliable means? When does it fail this check.

Reply:
Is this better?

```suggestion
|      | Reliability            | The tool operates consistently and reliably, meaning it delivers the same expected results every time you use it.             | M        |
```